packages/components/src/tools-panel/styles.ts

```diff
@@ -102,6 +102,11 @@ export const ToolsPanelItem = css`
 	/* Remove BaseControl components margins and leave spacing to grid layout */
 	&& ${ BaseControlWrapper } {
 		margin-bottom: 0;
+		/* The following allows controls with help text to maintain spacing. */
+		/* e.g. ToggleControls. */
+		display: flex;
```

Comment:
Is it "safe" to assume that _all_ control wrappers can adopt a flex column layout?

Reply:
Good point. I've pushed an alternative approach that I think still ticks the required boxes. https://github.com/WordPress/gutenberg/pull/36334/commits/9f9c0220af4885c3b6792dc14f1f1a05c0d7614d